### PR TITLE
Initial documentation for integration with react-native-vector-icons

### PR DESCRIPTION
**Pull request by zzorba**
As requested, a small writeup detailing how I incorporated react-native-vector-icons into my React Native Navigation project.

### CODE REVIEW

File: website/docs/docs/third-party-react-native-vector-icons.mdx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+## Using SVG icons in BottomTabs or TopBarButtons
+
+In the example below we will detail a simple app that pre-loads several icons from react-native-vector-icons to be used in our navigation. The example assumes
+you have worked through the [installation](https://github.com/oblador/react-native-vector-icons#installation) and optional steps to support [custom fonts](https://github.com/oblador/react-native-vector-icons#custom-fonts)
+according to the react-native-vector-icons project.
+
+### Step 1 - Choose icons and load image source.
+
+The core issue here is that React Native Navigation wants 'images' for its icons while react-native-vector-icon typically works as a component. However,
+there is a way around this, by using the `getImageSource` method exposed by react-native-vector-icons.
```
Comment (danilobuerger) [inline website/docs/docs/third-party-react-native-vector-icons.mdx:20]: There seems to also be `getImageSourceSync`. Why not use that instead?

Comment (zzorba) [reply website/docs/docs/third-party-react-native-vector-icons.mdx:20]: Yeah -- not totally sure (I wrote my code several years ago, so possible the API was extended, or maybe I just didn't use it before).

Let me try out the sync option and see how that works.

Comment (a-eid) [reply website/docs/docs/third-party-react-native-vector-icons.mdx:20]: `getImageSource` returns a promise, you have to either await it or use `getImageSourceSync`

Comment (zzorba) [reply website/docs/docs/third-party-react-native-vector-icons.mdx]: Okay, switched the example to use `getImageSourceSync` instead, which seems to work.

**Comment by guyca** [comment]
LGTM!

**Review by yogevbd — APPROVED** [review]

**Pull request merged**